docs/src/hugo/content/standards.md

```diff
@@ -0,0 +1,90 @@
+---
+type: common
+menu: main
+weight: 150
+title: Coding Standard
+---
+
+## Types
+
+### Effects
+
+Prefer a parameterized effect type over specializing on a task.
+
+```scala
+// Good
+def apply[F[_]](service: HttpService[F])(implicit F: Monad[F]): HttpService[F]
```

Comment:
Doesnt this apply to 0.18 or after effect is more presently, we are hard-coded to `Task` .